src/textual/widgets/_markdown.py

```diff
@@ -874,7 +874,7 @@ def table_of_contents(self) -> MarkdownTableOfContents:
 
     async def on_mount(self) -> None:
```

Comment:
Make method non-async?

```suggestion
    def on_mount(self) -> None:
```